src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -672,23 +664,13 @@ TEST_F(SetGlobalPropertiesRequestTest, Run_NoVRNoDataDefaultCreated_SUCCESS) {
       .WillOnce(Return(mock_app_));
   EXPECT_CALL(mock_message_helper_, VerifyImageVrHelpItems(_, _, _)).Times(0);
   EXPECT_CALL(app_mngr_, RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
-  SmartObject vr_help_title(smart_objects::SmartType_Null);
-  EXPECT_CALL(*mock_app_, vr_help_title())
-      .Times(2)
-      .WillRepeatedly(Return(&vr_help_title));
+  EXPECT_CALL(*mock_app_, vr_help_title()).Times(0);
 
-  CommandsMap commands_map;
-  SmartObject command_text(smart_objects::SmartType_Map);
-  commands_map[0] = &command_text;
-  (*commands_map[0])[am::strings::vr_commands] = SmartObject("one");
-  DataAccessor<CommandsMap> accessor(commands_map, lock_);
-  EXPECT_CALL(*mock_app_, commands_map()).WillOnce(Return(accessor));
-  EXPECT_CALL(*mock_app_, set_vr_help(_));
-  const CustomString name("name");
-  EXPECT_CALL(*mock_app_, name()).WillOnce(ReturnRef(name));
-  EXPECT_CALL(*mock_app_, set_vr_help_title(SmartObject(name)));
-  SmartObject vr_help_array(smart_objects::SmartType_Array);
-  EXPECT_CALL(*mock_app_, vr_help()).WillOnce(Return(&vr_help_array));
+  EXPECT_CALL(*mock_app_, commands_map()).Times(0);
```

Comment:
@SNiukalov you removed this functionality from implementation so you could remove these expectations from unit test as well